openmls/src/messages/mod.rs

```diff
@@ -242,7 +241,7 @@ impl Signable for GroupInfoTBS {
 ///     opaque signature<V>;
 /// } GroupInfo;
 /// ```
-pub(crate) struct GroupInfo {
+pub struct GroupInfo {
```

Comment:
`GroupInfo` is now `pub`.

Reply:
Right, since it's required for joining by external commits it needs to be public.